actors.md

```diff
@@ -658,23 +658,23 @@ func SubmitPost(proofs PoStProof, faults [FaultSet], recovered Bitfield, done Bi
 
 	feesRequired := 0
 
-	if chain.Now() > miner.ProvingPeriodEnd+GenerationAttackTime(miner.SectorSize) {
+	if chain.Now() > self.ProvingPeriodEnd+GenerationAttackTime(self.SectorSize) {
         // slashing ourselves
         SlashStorageFault(self)
         return
-	} else if chain.Now() > miner.ProvingPeriodEnd {
-		feesRequired += ComputeLateFee(miner.Power, chain.Now()-miner.ProvingPeriodEnd)
+	} else if chain.Now() > self.ProvingPeriodEnd {
+		feesRequired += ComputeLateFee(miner.power, chain.Now()-self.provingPeriodEnd)
```

Comment:
`provingPeriodEnd` is not defined on the actor state, can you add that?

Reply:
It's on line 490